### PR TITLE
added prebuild image for tiny calorie + docker compose

### PR DESCRIPTION
**Pull request by hra42**
1. I prebuild the tiny calorie dockerimage and published it on docker hub
2. added a simple docker-compose file for easy integration with something like portainer
3. worte a fully integrated version into my authelia and traefik stack

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,76 @@ This will expose it on port 8080, and store the SQLite db.db in /path/to/storage
...
+
+```
+version: "3.1"
+
+services:
+  tiny-calorie:
+    image: dnshra/tinycalorie:1.0
+    container_name: tiny-calorie
+    volumes: 
+      - /path/to/storage:/app/data
```
Comment (Uriopass) [inline README.md]: The db must be stored in `/storage` or it won't be synchronized to the volume.

Neither `/path/to/storage` or `/app/data` will work here.

Same goes for the other places you put `/app/data`, I think you got the order wrong.
Isn't the syntax
`/hostpath:/containerpath` ?

**Review by Uriopass — CHANGES_REQUESTED** [review]

**Comment by Uriopass** [comment]
Thank you for the PR ! I should try to figure out dockerhub and try to upload an "official" image though :D But it's not blocking in the meantime.

File: README.md (hunk 1 above)
Comment (hra42) [reply README.md]: no you are right, was copied from my template, so I need to add the correct path :D

**Comment by hra42** [comment]
corrected the path on readme file and the seperate compose files

the /path/to/storage need to be edited to comply with the local setup of the user

In my case that would be /dockerdata/tinycalories

**Comment by Uriopass** [comment]
LGTM ! Thanks.

**Pull request merged**